src/jdk.jpackage/windows/classes/jdk/jpackage/internal/WinExeBundler.java

```diff
@@ -141,6 +141,8 @@ private Path buildEXE(Map<String, ? super Object> params, Path msi,
 
         Files.copy(exePath, dstExePath);
 
+        dstExePath.toFile().setWritable(true, true);
```

Comment:
The jlink code in DefaultImageBuilder might be useful if you need finer control on the file permissions.